packages/lit-protocol-cipher/README.md

```diff
@@ -0,0 +1,22 @@
+# @requestnetwork/lit-protocol-cipher
+
+Lit Protocol Provider.
+
+`@requestnetwork/lit-protocol-cipher` is a typescript library part of the [Request Network protocol](https://github.com/RequestNetwork/requestNetwork).
+
+## Installation
+
+```bash
+npm install @requestnetwork/lit-protocol-cipher
+```
+
+## Usage
+
```

Comment:
The `Usage` section is emtpy, can we add more information.

Reply:
Added it